va/va.go

```diff
@@ -733,11 +807,18 @@ func (va *ValidationAuthorityImpl) PerformValidation(ctx context.Context, req *v
 		return bgrpc.ValidationResultToPB(records, filterProblemDetails(prob))
 	}
 
+	// If this is a remote VA (indicated by no backends), we can return now.
+	if len(va.remoteVAs) == 0 {
+		return bgrpc.ValidationResultToPB(records, nil)
+	}
+
 	// Do remote validation. We do this after local validation is complete to
 	// avoid wasting work when validation will fail anyway. This only returns a
 	// singular problem, because the remote VAs have already audit-logged their
 	// own validation records, and it's not helpful to present multiple large
 	// errors to the end user.
-	prob = va.performRemoteValidation(ctx, req)
+	mpicSummary, prob := va.performRemoteValidation(ctx, req)
+	logEvent.MPICSummary = mpicSummary
+
 	return bgrpc.ValidationResultToPB(records, filterProblemDetails(prob))
```

Comment:
bgrpc.ValidationResultToPB must be modified to pass through the RIR and Perspective or the Primary VA will just get back empty strings.

Reply:
Fixed in https://github.com/letsencrypt/boulder/pull/7828